lib/sterling/common/Request.js

```diff
@@ -0,0 +1,41 @@
+const request = require('../../common/Request');
+const { Handler, ApiErrors, ModuleError } = require('../../common/utils/errors');
+
+exports.trigger = async({
+    method = 'POST', path, credentials, payload,
+}) => {
+    const baseURL = credentials.host || 'https://sandboxapi.fsi.ng';
```

Comment:
Let's move baseURL, common request options to base `Request.http`

Reply:
noted